extensions/amp-story/1.0/amp-story.js

```diff
@@ -1088,6 +1116,16 @@ export class AmpStory extends AMP.BaseElement {
     }, {});
   }
 
+  /**
+   * Reacts to the browser tab becoming active/inactive.
+   * @private
+   */
+  onVisibilityChanged_() {
+    this.documentState_.isHidden() ?
```

Comment:
I thought `pauseCallback` and `resumeCallback` were automatically called when the document was hidden/shown?

Reply:
Only when the component is too far away from the user's viewport, or cases when it's in a carousel pan, as far as I know... Not on tab change. It's tracked here: #16795